include/zephyr/net/wifi_native.h

```diff
@@ -0,0 +1,119 @@
+/** @file
+ * @brief Native Wi-Fi API
+ *
+ * This file contains the native Wi-Fi API. These APIs are used by the
+ * application to control the Wi-Fi device directly without going through
+ * offloaded Wi-Fi APIs.
+ */
+
+/*
+ * Copyright (c) Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_ZEPHYR_NET_WIFI_NATIVE_H_
+#define ZEPHYR_INCLUDE_ZEPHYR_NET_WIFI_NATIVE_H_
+
+#include <zephyr/kernel.h>
+#include <zephyr/types.h>
+#include <zephyr/net/net_if.h>
+#include <zephyr/net/wifi_mgmt.h>
+/**
+ * @brief Native Wi-Fi API
+ * @defgroup wifi_native Native Wi-Fi API
+ * @ingroup networking
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Wi-Fi Native Managed Interface
+ */
+struct wifi_native_mgd_iface {
+	/** Interface */
+	struct net_if *iface;
+	/** Wi-Fi Management operations */
+	const struct wifi_mgmt_ops *ops;
```

Comment:
Shouldn't `struct wifi_mgmt_ops` be rather part of `struct wifi_native_stack`? My understanding is that we want to share a common implementation across all managed interfaces?

Reply:
Yes, I can move it there.